redirect/routes.php

```diff
@@ -262,20 +266,34 @@ class Route {
         ],
     ];
 
-    public function getTargetOfSub($subdomain) {
+    public function getTargetOfSub($subdomain, $subsub) {
```

Comment:
`$subsub` there must be a better name for this

Reply:
Nah.

Reply:
what about renaming `$subdomain` to `$course` and `$subsub` to `$site`

Reply:
Course is not really correct in all contexts. Maybe `$redirectUrl`

`$siteType` would work.